operations/boot_orchestration/BOS_API_Versions.md

```diff
@@ -0,0 +1,41 @@
+# Boot Orchestration
+
+The Boot Orchestration Service \(BOS\) currently supports two API versions, v1 and v2, that have different APIs and underlying mechanisms for performing operations on nodes.
+The following is a summary of the changes, and the upgrade path, for users wishing to compare the two.
+
+## BOS v2 improvements
+
+BOS v2 makes significant improvements to boot times, retries, and error handling, by allowing nodes to proceed through the boot process at their own pace.
+Nodes that require extra time or retries to complete will no longer hold up the rest of the nodes, meaning that more nodes will reach a ready state faster.
+
+## API Differences
+
+The v2 endpoints have plural nouns, using `sessions` and `sessiontemplates` rather than `session` and `sessiontemplate`.  This is more consistent with our other APIs.
```

Comment:
```suggestion
The v2 endpoints have plural nouns, using `sessions` and `sessiontemplates` rather than `session` and `sessiontemplate`. This is more consistent with other CSM APIs.
```